src/main/java/fi/vm/yti/terminology/api/importapi/ImportService.java

```diff
@@ -223,12 +223,39 @@ ResponseEntity<String> handleNtrfDocumentAsync(String format, UUID vocabularyId,
             System.out.println("Incoming transform error=" + ioe);
         } catch (XMLStreamException se) {
             System.out.println("Incoming transform error=" + se);
-        } catch(JAXBException je){
+        } catch (JAXBException je){
             System.out.println("Incoming transform error=" + je);
         }
         return new ResponseEntity<>( rv, HttpStatus.OK);
     }
 
+    public UUID handleSimpleExcelImport(UUID terminologyId, InputStream is) throws NullPointerException, IOException {
+        boolean exists = terminologyExists(terminologyId);
```

Comment:
Should we check that user can modify terminology before reading the entire excel file? This is not such a problem in the UI since the import button wont be visible if they dont have the permission to modify. but might be frustrating if they upload a file with a lot of rows

Reply:
That might be good idea, since the terminology id is provided in the url